mifosng-android/src/main/java/com/mifos/api/datamanager/DataManagerSharing.java

```diff
@@ -0,0 +1,64 @@
+package com.mifos.api.datamanager;
+
+import com.mifos.api.BaseApiManager;
+import com.mifos.objects.response.ShareResponse;
+import com.mifos.objects.templates.sharing.SharingProductTemplate;
+import com.mifos.objects.templates.sharing.SharingAccountsTemplate;
+import com.mifos.services.data.SharingPayload;
+
+import javax.inject.Inject;
+
+import rx.Observable;
+
+/**
+ * Created by mayankjindal on 18/08/17.
+ */
+/**
+ * This DataManager is for Managing Client's Share Account, In which Request is going to Server
+ * and In Response, We are getting Client's Share Account Observable Response using Retrofit2 .
+ */
+public class DataManagerSharing {
+
+    public final BaseApiManager mBaseApiManager;
```

Comment:
I know at many places we are using `m` convention in naming but we no longer supporting this convention please rename the variable `baseApiManager`

Reply:
Done.